tests/Silverback.Integration.Tests.E2E/Kafka/EncryptionTests.cs

```diff
@@ -84,6 +84,183 @@ public async Task Encryption_SimpleMessages_EncryptedAndDecrypted()
             DefaultTopic.GetCommittedOffsetsCount("consumer1").Should().Be(2);
         }
 
+        [Fact]
+        public async Task Encryption_SimpleMessagesWithKeyRotation_EncryptedAndDecrypted()
```

Comment:
Am I wrong or this test in the end just uses the same key for both messages? Isn't worth testing with two different keys, just to be sure it actually works?

Reply:
You're right! I modified the test and now two different keys are taken in account.